addons_pos/pos_auto_reconcile/models/pos_session.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+
+from openerp import models, fields, api, _
+import logging
+
+_logger = logging.getLogger(__name__)
+
+
+class POSSession(models.Model):
+    _name = 'pos.session'
```

Comment:
Redefinition of the ```_name``` is only needed if it differs...make it easier to read and distinguish between style of inheritance and remove it.